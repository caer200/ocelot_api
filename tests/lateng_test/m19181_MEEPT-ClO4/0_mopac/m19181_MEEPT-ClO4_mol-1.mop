CHARGES

 
H	6.029753 +1 0.218912 +1 2.793611 +1
C	5.906688 +1 0.743143 +1 2.010976 +1
C	5.538913 +1 0.110419 +1 0.801142 +1
C	6.088486 +1 2.104305 +1 2.073299 +1
C	5.365543 +1 0.943191 +1 -0.332668 +1
N	5.373832 +1 -1.271660 +1 0.741624 +1
H	6.345984 +1 2.503374 +1 2.896136 +1
C	5.901115 +1 2.912990 +1 0.945604 +1
C	5.545182 +1 2.330215 +1 -0.246969 +1
S	5.005082 +1 0.355979 +1 -1.915553 +1
C	4.956533 +1 -1.966883 +1 -0.391366 +1
C	5.772951 +1 -2.055751 +1 1.944608 +1
H	6.018149 +1 3.853950 +1 1.004272 +1
H	5.420041 +1 2.870977 +1 -1.017873 +1
C	4.790059 +1 -1.332004 +1 -1.647943 +1
C	4.682164 +1 -3.353399 +1 -0.321970 +1
H	6.555734 +1 -1.620565 +1 2.366021 +1
H	6.049047 +1 -2.962609 +1 1.659388 +1
C	4.667537 +1 -2.180622 +1 2.974040 +1
C	4.441857 +1 -2.075908 +1 -2.787315 +1
H	4.759160 +1 -3.807462 +1 0.508931 +1
C	4.304638 +1 -4.048496 +1 -1.445102 +1
H	5.021660 +1 -2.591418 +1 3.802346 +1
H	4.311213 +1 -1.284591 +1 3.198133 +1
O	3.638882 +1 -2.989561 +1 2.434092 +1
H	4.373575 +1 -1.642674 +1 -3.629935 +1
C	4.198763 +1 -3.421350 +1 -2.693746 +1
H	4.110880 +1 -4.975815 +1 -1.373532 +1
C	2.749953 +1 -3.511232 +1 3.420928 +1
H	3.961604 +1 -3.922420 +1 -3.465187 +1
H	2.089059 +1 -2.821042 +1 3.679926 +1
H	3.258054 +1 -3.774945 +1 4.228679 +1
C	2.048534 +1 -4.700996 +1 2.847222 +1
H	2.709289 +1 -5.320460 +1 2.447925 +1
H	1.564311 +1 -5.183342 +1 3.563676 +1
O	1.127146 +1 -4.266292 +1 1.848619 +1
C	0.739729 +1 -5.344243 +1 1.019674 +1
H	0.412855 +1 -6.082467 +1 1.575430 +1
H	1.510043 +1 -5.646444 +1 0.495091 +1
H	0.026817 +1 -5.050953 +1 0.414389 +1
