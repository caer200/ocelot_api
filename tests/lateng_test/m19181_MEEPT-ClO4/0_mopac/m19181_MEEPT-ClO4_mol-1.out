 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:32:31 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          6.02975300  *   0.21891200  *   2.79361100  *
     2       C          5.90668800  *   0.74314300  *   2.01097600  *
     3       C          5.53891300  *   0.11041900  *   0.80114200  *
     4       C          6.08848600  *   2.10430500  *   2.07329900  *
     5       C          5.36554300  *   0.94319100  *  -0.33266800  *
     6       N          5.37383200  *  -1.27166000  *   0.74162400  *
     7       H          6.34598400  *   2.50337400  *   2.89613600  *
     8       C          5.90111500  *   2.91299000  *   0.94560400  *
     9       C          5.54518200  *   2.33021500  *  -0.24696900  *
    10       S          5.00508200  *   0.35597900  *  -1.91555300  *
    11       C          4.95653300  *  -1.96688300  *  -0.39136600  *
    12       C          5.77295100  *  -2.05575100  *   1.94460800  *
    13       H          6.01814900  *   3.85395000  *   1.00427200  *
    14       H          5.42004100  *   2.87097700  *  -1.01787300  *
    15       C          4.79005900  *  -1.33200400  *  -1.64794300  *
    16       C          4.68216400  *  -3.35339900  *  -0.32197000  *
    17       H          6.55573400  *  -1.62056500  *   2.36602100  *
    18       H          6.04904700  *  -2.96260900  *   1.65938800  *
    19       C          4.66753700  *  -2.18062200  *   2.97404000  *
    20       C          4.44185700  *  -2.07590800  *  -2.78731500  *
    21       H          4.75916000  *  -3.80746200  *   0.50893100  *
    22       C          4.30463800  *  -4.04849600  *  -1.44510200  *
    23       H          5.02166000  *  -2.59141800  *   3.80234600  *
    24       H          4.31121300  *  -1.28459100  *   3.19813300  *
    25       O          3.63888200  *  -2.98956100  *   2.43409200  *
    26       H          4.37357500  *  -1.64267400  *  -3.62993500  *
    27       C          4.19876300  *  -3.42135000  *  -2.69374600  *
    28       H          4.11088000  *  -4.97581500  *  -1.37353200  *
    29       C          2.74995300  *  -3.51123200  *   3.42092800  *
    30       H          3.96160400  *  -3.92242000  *  -3.46518700  *
    31       H          2.08905900  *  -2.82104200  *   3.67992600  *
    32       H          3.25805400  *  -3.77494500  *   4.22867900  *
    33       C          2.04853400  *  -4.70099600  *   2.84722200  *
    34       H          2.70928900  *  -5.32046000  *   2.44792500  *
    35       H          1.56431100  *  -5.18334200  *   3.56367600  *
    36       O          1.12714600  *  -4.26629200  *   1.84861900  *
    37       C          0.73972900  *  -5.34424300  *   1.01967400  *
    38       H          0.41285500  *  -6.08246700  *   1.57543000  *
    39       H          1.51004300  *  -5.64644400  *   0.49509100  *
    40       H          0.02681700  *  -5.05095300  *   0.41438900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          6.0298      0.2189      2.7936
     2         C          5.9067      0.7431      2.0110
     3         C          5.5389      0.1104      0.8011
     4         C          6.0885      2.1043      2.0733
     5         C          5.3655      0.9432     -0.3327
     6         N          5.3738     -1.2717      0.7416
     7         H          6.3460      2.5034      2.8961
     8         C          5.9011      2.9130      0.9456
     9         C          5.5452      2.3302     -0.2470
    10         S          5.0051      0.3560     -1.9156
    11         C          4.9565     -1.9669     -0.3914
    12         C          5.7730     -2.0558      1.9446
    13         H          6.0181      3.8540      1.0043
    14         H          5.4200      2.8710     -1.0179
    15         C          4.7901     -1.3320     -1.6479
    16         C          4.6822     -3.3534     -0.3220
    17         H          6.5557     -1.6206      2.3660
    18         H          6.0490     -2.9626      1.6594
    19         C          4.6675     -2.1806      2.9740
    20         C          4.4419     -2.0759     -2.7873
    21         H          4.7592     -3.8075      0.5089
    22         C          4.3046     -4.0485     -1.4451
    23         H          5.0217     -2.5914      3.8023
    24         H          4.3112     -1.2846      3.1981
    25         O          3.6389     -2.9896      2.4341
    26         H          4.3736     -1.6427     -3.6299
    27         C          4.1988     -3.4213     -2.6937
    28         H          4.1109     -4.9758     -1.3735
    29         C          2.7500     -3.5112      3.4209
    30         H          3.9616     -3.9224     -3.4652
    31         H          2.0891     -2.8210      3.6799
    32         H          3.2581     -3.7749      4.2287
    33         C          2.0485     -4.7010      2.8472
    34         H          2.7093     -5.3205      2.4479
    35         H          1.5643     -5.1833      3.5637
    36         O          1.1271     -4.2663      1.8486
    37         C          0.7397     -5.3442      1.0197
    38         H          0.4129     -6.0825      1.5754
    39         H          1.5100     -5.6464      0.4951
    40         H          0.0268     -5.0510      0.4144

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
